### DISCUSSION TITLE
What is the Webhook Secret in AI Settings?

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#2644] [Q&A]
when you enable the AI service it asks for a webhook secret. This is not mentioned in your docs. Is this a new webhook secret we need to generate on our own, or this the same webhook secret that is in the Secret settings (i.e. NHOST_WEBHOOK_SECRET)? Do these 2 secrets need to be equal?

**Comment by dbarrosop** [comment]
It is for internal use of hasura and graphite. You can set it to anything you want and it doesn't have to be related at all with the `NHOST_WEBHOOK_SECRET`. I will add a note to clarify that in the dashboard. Thanks!
